### DISCUSSION TITLE
What is the meaning of error status info:0?

### DISCUSSION THREAD

**Discussion by RockyHoo1209** [PaddlePaddle/PaddleSpatial#89] [BaiduKDDCup2022] [answered]
What is the meaning of error status info:0?
![image](https://user-images.githubusercontent.com/41967408/169650426-35a52e91-ac2c-406c-a95d-4989c3b5f79b.png)

**Comment by ramber1836** [comment, marked as answer]
You can provide your team name and we will check the backend log.
Due to the hard workload, the reply might not be as soon as you expected.
Thanks for your understanding.

Regards,
Xinjiang Lu

**Reply by RockyHoo1209** [reply]
I have found out the reason,thanks!
